src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -26,13 +27,19 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
-    List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).list();
-    ArrayList<UserData> mentors = new ArrayList<UserData>();
-    for (UserEntity entity : mentorEntities) {
-      if (entity.isMentor()) {
-        mentors.add(new UserData(entity));
-      }
-    }
+    addMockMentor(response); // TODO: remove by production
+    List<UserData> mentors =
+        ofy().load().type(UserEntity.class).filter("isMentor", true).list().stream()
+            .map(
+                userEntity -> {
+                  try {
+                    return new UserData(userEntity);
+                  } catch (IOException e) {
+                    return null;
+                  }
+                })
+            .collect(Collectors.toList());
```

Comment:
@josephdhurst does this fix your previous issue?

Reply:
```suggestion
    List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).filter("isMentor", true).list();
    ArrayList<UserData> mentors = new ArrayList<UserData>();
    for (UserEntity entity : mentorEntities) {
      mentors.add(new UserData(entity));
    }
```